app/assets/stylesheets/_fonts.scss

```diff
@@ -0,0 +1,55 @@
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 300;
+  src: font-url("sourcesanspro-light-webfont.eot?#iefix") format("embedded-opentype"), font-url("sourcesanspro-light-webfont.woff2") format("woff2"), font-url("sourcesanspro-light-webfont.woff") format("woff"), font-url("sourcesanspro-light-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 400;
+  src: font-url("sourcesanspro-regular-webfont.eot?#iefix") format("embedded-opentype"), font-url("sourcesanspro-regular-webfont.woff2") format("woff2"), font-url("sourcesanspro-regular-webfont.woff") format("woff"), font-url("sourcesanspro-regular-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: italic;
+  font-weight: 400;
+  src: font-url("sourcesanspro-italic-webfont.eot?#iefix") format("embedded-opentype"), font-url("sourcesanspro-italic-webfont.woff2") format("woff2"), font-url("sourcesanspro-italic-webfont.woff") format("woff"), font-url("sourcesanspro-italic-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 700;
+  src: font-url("sourcesanspro-bold-webfont.eot?#iefix") format("embedded-opentype"), font-url("sourcesanspro-bold-webfont.woff2") format("woff2"), font-url("sourcesanspro-bold-webfont.woff") format("woff"), font-url("sourcesanspro-bold-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Merriweather";
+  font-style: normal;
+  font-weight: 300;
+  src: font-url("merriweather-light-webfont.eot?#iefix") format("embedded-opentype"), font-url("merriweather-light-webfont.woff2") format("woff2"), font-url("merriweather-light-webfont.woff") format("woff"), font-url("merriweather-light-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Merriweather";
+  font-style: normal;
+  font-weight: 400;
+  src: font-url("merriweather-regular-webfont.eot?#iefix") format("embedded-opentype"), font-url("merriweather-regular-webfont.woff2") format("woff2"), font-url("merriweather-regular-webfont.woff") format("woff"), font-url("merriweather-regular-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Merriweather";
+  font-style: italic;
+  font-weight: 400;
+  src: font-url("merriweather-italic-webfont.eot?#iefix") format("embedded-opentype"), font-url("merriweather-italic-webfont.woff2") format("woff2"), font-url("merriweather-italic-webfont.woff") format("woff"), font-url("merriweather-italic-webfont.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Merriweather";
+  font-style: normal;
+  font-weight: 700;
+  src: font-url("merriweather-bold-webfont.eot?#iefix") format("embedded-opentype"), font-url("merriweather-bold-webfont.woff2") format("woff2"), font-url("merriweather-bold-webfont.woff") format("woff"), font-url("merriweather-bold-webfont.ttf") format("truetype");
+}
```

Comment:
Just a minor style issue that Code Climate would have flagged if the webhook was working: Please convert the double quotes to single quotes.